control-plane-operator/controllers/hostedcontrolplane/oapi/config.go

```diff
@@ -58,8 +58,8 @@ func reconcileConfigObject(cfg *openshiftcpv1.OpenShiftAPIServerConfig, auditWeb
 	cfg.APIServerArguments = map[string][]string{
 		"shutdown-delay-duration": {"15s"},
 		"audit-log-format":        {"json"},
-		"audit-log-maxsize":       {"100"},
-		"audit-log-maxbackup":     {"10"},
+		"audit-log-maxsize":       {"10"},
+		"audit-log-maxbackup":     {"0"},
```

Comment:
Per https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/, `The maximum number of old audit log files to retain. Setting a value of 0 will mean there's no restriction on the number of files.`  ... I think we will have an unlimited number of backups.  A quick review of the Kubernetes code seems to confirm this.